compose/services/docker-compose.redis-admin.yml

```
@@ -0,0 +1,14 @@
+    image: redis/redisinsight:latest
+    container_name: redisinsight
+    hostname: redisinsight
+    restart: unless-stopped
+    volumes:
+      - redis-insight:/data
+    ports:
+      - "8081:5540"
+
+    networks:
+      - internal
+      - web
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)